### PR TITLE
Test of java robot.

### PR DESCRIPTION
**Pull request by EliosPride**

### CODE REVIEW

File: src/test/java/core/basesyntax/FieldXYTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FieldXYTest {
+    @Test
```
Comment (al-pavlenko) [inline src/test/java/core/basesyntax/FieldXYTest.java:8]: @EliosPride add empty line

File: src/test/java/core/basesyntax/FieldXYTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    public void testMoveRobotWhenMoveDirectionUpXCoordinate() {
+        Robot robot = new Robot(Direction.UP, 0, 0);
+        FieldXY field = new FieldXY();
+        field.moveRobot(robot, 10, 0);
+        Assert.assertEquals(10, robot.getCoordinateX());
```
Comment (al-pavlenko) [inline src/test/java/core/basesyntax/FieldXYTest.java:13]: @EliosPride add assert on Y coordinate, the same for other tests

File: src/test/java/core/basesyntax/FieldXYTest.java (hunk 1 above)
Comment (EliosPride) [reply src/test/java/core/basesyntax/FieldXYTest.java:8]: @al-pavlenko done

File: src/test/java/core/basesyntax/FieldXYTest.java (hunk 2 above)
Comment (EliosPride) [reply src/test/java/core/basesyntax/FieldXYTest.java:13]: @al-pavlenko all done

**Pull request merged**